src/feature.rs

```diff
@@ -93,7 +93,7 @@ impl Feature {
             .and_then(|props| props.get(key.as_ref()))
     }
 
-    /// Return true iff this key is set
+    /// Return true if this key is set
```

Comment:
This isn't a typo, "iff" is often used for "if and only if".

Reply:
Oh! I didn't know that! PTAL again.